proposals/0355-regex-syntax-run-time-construction.md

```diff
@@ -573,7 +581,7 @@ Custom characters classes introduce their own sublanguage, in which most regular
 
 Atoms may be used to compose other character class members, including ranges, quoted sequences, and even nested custom character classes `[[ab]c\d]`. Adjacent members form an implicit union of character classes, e.g `[[ab]c\d]` is the union of the characters `a`, `b`, `c`, and digit characters.
 
-Custom character classes may not be empty, e.g `[]` is forbidden. A custom character class may begin with the `]` character, in which case it is treated as literal, e.g `[]a]` is the custom character class of `]` and `a`.
+Custom character classes may not be empty, e.g `[]` is forbidden.
 
 Quoted sequences may be used to escape the contained characters, e.g `[a\Q]\E]` is also the character class of `[` and `a`.
```

Comment:
I don’t believe “also” is needed with the changes to the previous line.

I incorporated the change from #1655 here as well, (`[` -> `]`). So that can be closed if this is merged.

```suggestion
Quoted sequences may be used to escape the contained characters, e.g `[a\Q]\E]` is the character class of `]` and `a`.
```

Reply:
Thanks!